### DISCUSSION TITLE
Task1: Is there a description of the model architecture?

### DISCUSSION THREAD

**Discussion by leon-maechler** [FETS-AI/Challenge#86] [Q&A] [answered]
Hi! 

thank you again for organizing this challenge!
I was wondering if there is a description of the model architecture somewhere?
In the code I could only find this 

`# this is an adaptation of fets.models.pytorch.brainmage.BrainMage 3dresunet`

Thanks!

**Comment by sarthakpati** [comment, marked as answer]
It is a standard U-Net with residual connections. You can find the exact model code here: https://github.com/FETS-AI/Algorithms/blob/master/fets/models/pytorch/pt_3dresunet/pt_3dresunet.py

This is also identical to the model we use via FeTS and GaNDLF: https://github.com/CBICA/GaNDLF/blob/master/GANDLF/models/unet.py

**Reply by leon-maechler** [reply]
Cool thank you for the quick answer!
